website/docs/developer/development-guide/code/postgres-tips.md

```diff
@@ -0,0 +1,69 @@
+---
+id: postgres-tips
+title: Postgres Paging
+description: Tips on writing efficient Postgres queries.
+---
+
+## Paging
+It will not always be feasible/desired to return all results at once.
+In this case, results should be returned one 'page' at a time, with cursors included to allow customers to easily request the next page.
+
+### Sample Table
+Code in the following sections will be based off of the following table:
+```
+CREATE TABLE "orca"."test_table"  ( 
+	"key0"	int4 NULL,
+	"key1"	int4 NULL,
+	"key2"	int4 NULL 
+	)
+GO
+ALTER TABLE "orca"."test_table"
+	ADD CONSTRAINT "test_uniqueness"
+	UNIQUE ("key0", "key1", "key2")
+```
+
+Adjust code as needed to match your table.
+
+### Page Retrieval
+The initial impulse for retrieving a page may be to `SKIP` the number of entries from previous pages.
+However, this requires the query to process all of the previous entries, and is therefore non-performant on large datasets.
+
+A better option is to apply a `WHERE` clause that picks up after the previous query.
+For example, following from a page where the final entry was `key0: 50, key1: 98, key2: 60`, retrieving the next page of size `100` would look like:
```

Comment:
Reword this slightly so it is clear that we are using these values for all the examples below. Maybe like the following

```
In the examples below we are going to assume the row we will paging from has the following unique values of `key0: 50, key1: 98, key2: 60`. In all instances we will be limiting the result to `100` records.
```

You may also want break this into 2 sub sections like forward paging and reverse or backward paging below to better organize paging thoughts.

Reply:
Added line regarding all examples.